tools/typst-preview-frontend/src/main.js

```diff
@@ -37,7 +37,7 @@ function retrieveWsArgs() {
 
     /// The string `ws://127.0.0.1:23625` is a placeholder
     /// Also, it is the default url to connect to.
-    let url = "ws://127.0.0.1:23625";
+    let url = "ws://" + window.location.host;
```

Comment:
not sure if this is correct

Reply:
maybe `window.location["host"]`?

Reply:
![image](https://github.com/user-attachments/assets/cbae769d-6660-4261-8623-3692514c2132)

works well on newest chrome

Reply:
you are right，let me think...

Reply:
fixed.
i've tested the fix.
![image](https://github.com/user-attachments/assets/d4931d50-e495-46aa-bd74-22e00a6f38f8)